internal/types/webhook_logs.go

```diff
@@ -15,6 +15,9 @@ type WebhookLog struct {
 }
 
 type WebhookLogMessage struct {
-	Header http.Header
-	Body   []byte
+	Header  http.Header
+	Body    []byte
+	Method  string `json:",omitempty"`
```

Comment:
Is this the same as: 
```suggestion
	Method  string `json:"-"`
```
?

Reply:
It's not: `-` would omit the field entirely, whereas we only want it omitted when the string is empty, hence `omitempty`. (Omitting it means that we don't waste database space in the response messages encoding the keys that will never be used.)

(This behaviour is [described in the documentation for `json.Marshal`](https://pkg.go.dev/encoding/json#Marshal).)